openapi.yaml

```
@@ -0,0 +1,390 @@
+openapi: 3.0.1
+info:
+  title: Self Defined
+  description: |
+    A modern dictionary about us. We define our words, but they don't define us. [Self-Defined](https://www.selfdefined.app/) seeks to provide more inclusive, holistic, and fluid definitions to reflect the modern world. The goal for [Self-Defined](https://www.selfdefined.app/) is to become the first modern dictionary that reflects our digital world, using the best aspects of technology while minimising the worst aspects of technology that can cause harm. [Self-Defined](https://www.selfdefined.app/) is successful if it supports greater, more nuanced understanding and tolerance of fellow humans through words.
+  termsOfService: https://www.selfdefined.app/terms-of-service
+  # contact:
+  #   email: apiteam@swagger.io
+  # license:
+  #   name: Apache 2.0
+  #   url: http://www.apache.org/licenses/LICENSE-2.0.html
+  version: 1.0.0
+externalDocs:
+  description: Find out more
+  url: https://github.com/tatianamac/selfdefined
+servers:
+- url: https://api.selfdefined.app/
+tags:
+- name: Word
+  description: Self-Defined Words
+  externalDocs:
+    description: Find out more
+    url: https://schemas.selfdefined.app/words
+components:
+  schemas:
+    Word:
+      allOf: 
+      - $ref: '#/components/schemas/Id'
+      - $ref: '#/components/schemas/NewWord'
+    FullWord:
+      allOf: 
+      - $ref: '#/components/schemas/Word'
+      - $ref: '#/components/schemas/Translations'
+      - $ref: '#/components/schemas/Synonyms'
+      - $ref: '#/components/schemas/Alternatives'
+      - $ref: '#/components/schemas/FurtherReadings'
+    ShortLink:
+      description: Shortlink terms which map to this word.
+      type: object
+      properties:
+        shortLinkText:
+          type: string
+        word:
+          $ref: '#/components/schemas/Link'
+      required:
+      - shortLink
```

Should this be shortLinkText to match the property
```suggestion
      - shortLinkText
```